website/content/docs/configuration/session-recording/create-storage-bucket.mdx

```diff
@@ -54,6 +54,24 @@ For more information about using session recording to audit user sessions, refer
    }
    ```
 
+### Boundary Workers Requirements
+
+[Session recording](/boundary/docs/configuration/session-recording) requires at least one PKI worker with access to local and remote storage.
+- A Boundary PKI worker with access to the AWS S3 storage bucket
+- A Boundary PKI worker with an accessible directory defined by `recording_storage_path` for storing in-progress session recordings. On session closure, a local session recording is moved to remote storage and deleted locally. For more details, refer to [PKI workers](/boundary/docs/configuration/worker/pki-worker#session-recording-hcp-ent).
+- A Boundary PKI worker with a minimum of 1 MB available disk space.
+- A Boundary PKI worker running on darwin, windows or linux. The following binaries are not supported for session recording: nestbsd, openbsd, solaris
```

Comment:
```suggestion
[Session recording](/boundary/docs/configuration/session-recording) requires at least one PKI worker that:

- Has access to the AWS S3 storage bucket
- Has an accessible directory defined by `recording_storage_path` for storing session recordings while they are in progress. On session closure, Boundary moves the local session recording to remote storage and deletes the local copy. For more details, refer to [PKI workers](/boundary/docs/configuration/worker/pki-worker#session-recording-hcp-ent).
- Has at least 1 MB of available disk space.
- Runs Darwin, Windows, or Linux. The following binaries are not supported for session recording: NetBSD, OpenBSD, Solaris.
```

Reply:
I tried to consolidate this bulleted list so we don't have to repeat "A Boundary PKI worker" at the beginning of each line. But please make sure I didn't change the meaning.